p2panda-net/examples/chat.rs

```diff
@@ -55,7 +51,7 @@ async fn main() -> Result<()> {
     });
 
     println!(".. waiting for peers to join ..");
-    let _ = ready_rx.recv().await;
+    let _ = ready.await;
```

Comment:
The `tokio` oneshot receiver doesn't have a `recv()` method; it's a future that we can `await` directly.